src/kili/services/export/format/coco/__init__.py

```diff
@@ -63,7 +64,9 @@ class CocoExporter(AbstractExporter):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.download_media = True  # we need the assets for COCO annotation format
+        if not self.download_media:
+            warnings.warn("For COCO export, download_media must be set to True.")
```

Comment:
`"For the export to the COCO format, the download of assets cannot be disabled"`
`download_media` is not the parameter name in the CLI.